src/app/manage/ApplicationSettings.ts

```diff
@@ -81,6 +81,18 @@ export class ApplicationSettings extends Entity<number>  {
       this.validateSmsContent(this.reminderSmsText);
     }
   });
+  donorEmailText = new StringColumn({
+    caption: this.lang.emailDonorContentCaption,
+    validate: () => {
+      this.validateSmsContent(this.donorEmailText);
+    }
+  });
+  helperEmailText = new StringColumn({
```

Comment:
It's not clear from the name when is this beeing sent - the name should indicate that - something like "registerHelperReplyEmailText"